tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -115,6 +115,11 @@ def Runs(self):
                 self._reader.update()
                 # TODO(cais): Start off a reading thread here, instead of being
                 # called only once here.
+            except AttributeError as error:
+                # Gracefully fail for users without the required API changes to
+                # debug_events_reader introduced in TF 2.1.0.dev20200103.
```

Comment:
nit: debug_events_reader.DebugDataReader to be precise.

Reply:
Done.